articles/communication-services/how-tos/cte-calling-sdk/includes/manage-calls/manage-calls-web.md

```diff
@@ -342,10 +342,26 @@ Consider the following scenario, where Alice makes a call to Bob, then Alice add
 If Teams user stops call recording, the recording is placed into the chat associated with the thread. Provided chat ID impacts the experience of Teams users in Teams clients.
 
 Recommendations for the management of chat ID:
-- Escalation of the 1:1 phone call by adding another phone participant: Use Graph API to get the existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000"
-- Group call with single Teams user and multiple phone participants: Use Graph API to get existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000"
-- Group call with more than 2 Teams users: Use Graph API to get or create a group chat with the Teams users
-
+- Escalation of the 1:1 phone call by adding another phone participant:
+  * (Optional way) Use addParticipant API on web calling SDK to add another participant without providing chat thread ID. You only need to provide identifier of the participant you want to add into the call.
+    ```js
+    addParticipant(participant: MicrosoftTeamsUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsAppIdentifier | UnknownIdentifier)
+    ```
+  * Use Graph API to get the existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000".
+- Group call with single Teams user and multiple phone participants:
+  * (Optional way) Use startCall API on web calling SDK without providing chat thread ID to create a group chat for participants. You only need to provide identifier for all the participants when leveraging API.
+    ```js
+    startCall(MicrosoftTeamsUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsAppIdentifier | UnknownIdentifier)[])
+    ```
+  * Use Graph API to get existing chat ID with only Teams user as a participant or create a new group chat with participants: Teams user ID and "00000000-0000-0000-0000-000000000000".
+- Group call with more than 2 Teams users:
+  * (Optional way) Use startCall API on web calling SDK without providing chat thread ID to create a group chat for participants. You only need to provide identifier for all the participants when leveraging API.
+    ```js
+    startCall(MicrosoftTeamsUserIdentifier | PhoneNumberIdentifier | MicrosoftTeamsAppIdentifier | UnknownIdentifier)[])
+    ```
+  * Use Graph API to get or create a group chat with the Teams users.
+> [!NOTE]
+> If you are using optional way to leverage API without providing chat thread Id, then you need to be consistently using this way to let web calling SDK to create/manage the thread ID for your App. In this way, You can access thread ID via TeamsCallInfo object on web calling SDK.
```

Comment:
What does it mean? What if there is case where I want to create new thread but then other when I want to reuse thread ID? What mistakes can developer do? How do we prevent those mistakes? What are the error messages we are giving to them?

Reply:
> What does it mean? What if there is case where I want to create new thread but then other when I want to reuse thread ID? What mistakes can developer do? How do we prevent those mistakes? What are the error messages we are giving to them?

If the user wants to resuse an existing threadid , then they can provide the threadid in the startgroupcall options.
If the user wants the thread creation to be handled by the webcalling sdk then they will not provide startgroupcall options where thread is optional. In that case threadid is not provided.
And share the error messages that we are giving in different scenarios for the user.

Reply:
@lpx1989 , @tomaschladek , let me know if writing this way helps the developer.

sceanrio1 : Create Thread on your own.
1. Use Graph API to create a group chat with the teams participants.
2. Pass the threadid in the startCall Api on web calling SDK to create a group chat for the participants.
3. When trying to add the participant to the group call, first update the thread with the new participant using Graph Api and then call add participant.

scenario 1 : Allowing API to create the threadId.
1. Use the startCall API on web calling SDK without providing the chat threadID to create a group chat for participants. You only need to provide identifier for all the participants when leveraging the API.
2. Newly created threadid can be accessed to the developer using the TeamsCallInfo.ThreadId property.
3. When trying to add the participant to the same group call, do not provide the chat threadid since it is already available in the TeamsCallInfo, SDK will be reading this automatically and adds the new participant to the chat thread.

scenario 3: Reusing the existing threadid for the group call
1.If there is already a threadid available for the participants, provide the threadid in the startcall options along with the identifiers of all the participants when leveraging the API.

Reply:
I think we are making it too complicated. This feature was supposed to simplify the developer experience.

1. Default behavior is that ACS calling SDK takes care of threads. This is not something we need to describe in details. 
2. Developer can provide optionally unique thread ID to start group call or add participant. This causes XYZ in ACS and XYZ in Teams app. You can manage chat threads via Graph API [link].

We should only provide note about second step and be clear about the implications. That is it.

Reply:
@tomaschladek , Please let me know if this is ok.

Group call with more than 2 Teams users: 
1. When making a group call with more than 2 Teams users using ACS Calling SDK, the SDK will automatically create the thread by default.
2.  If desired, the developer can provide a unique thread ID to start the group call or add participants. In this case, the ACS Calling SDK will use the given thread ID to create the group call.
A chat thread is created for the Teams users, which is visible in the Teams app for the participants. This allows them to chat during the call. 
Chat thread management can be done via [Graph Api ](https://learn.microsoft.com/en-us/graph/api/group-post-threads?view=graph-rest-1.0&tabs=http)

Reply:
That is already written in the documentation above, right? Then we don't need to create any new note.

How about this:
1. Let's use Microsoft 365 user instead of Teams user
2. `which is visible in the Teams app for the participants.` That is always the case, right? I think it should state: and this thread is associated to the group call for users in Teams app.  Let's fixt the statements in the sentences above.
3. Remove note as all info is already there.

I will review it and ideally this gets merged